airflow/providers/hashicorp/secrets/vault.py

```diff
@@ -108,7 +110,7 @@ def __init__(
         key_id: str | None = None,
         secret_id: str | None = None,
         role_id: str | None = None,
-        role_arn: str | None = None,
+        assume_role_kwargs: dict | None = None,
```

Comment:
This require also change to 
`docs/apache-airflow-providers-hashicorp/secrets-backends/hashicorp-vault.rst` isn't it?
 in https://github.com/apache/airflow/pull/38536/files it added 
` Add parameter "role_arn": "The AWS ARN of the role to assume"`

Reply:
Aha, sorry! let me fix this! thanks!